cordova-common/src/superspawn.js

```diff
@@ -22,7 +22,7 @@ var fs = require('fs');
 var path = require('path');
 var _ = require('underscore');
 var Q = require('q');
-var shell = require('shelljs');
+var which = require('shelljs/src/which');
```

Comment:
this eliminates multiple copies of `shelljs` - since every module gets it's own version. There are two versions of cordova-common loaded and hence two shelljs modules.

If we do update the version of shelljs and this breaks - it would be very obvious to fix. I have a PR in the works to submit to shelljs to delay load on their side.